scss/_variables.scss

```diff
@@ -1820,6 +1822,10 @@ $carousel-control-icon-width:        2.5rem !default;
 $carousel-control-icon-size:         1rem 1.5rem !default;
 $carousel-control-icon-bg:           var(--#{$boosted-prefix}chevron-icon) !default;
 $carousel-control-icon-active-bg:    $component-active-bg !default;
+
+$carousel-control-pause-icon:        $pause-icon !default;
+$carousel-control-play-icon:         $play-icon !default;
+$carousel-control-pause-icon-size:   .875rem .875rem !default;
```

Comment:
Is the size OK? I can observe a difference with the one in https://deploy-preview-1146--boosted.netlify.app/docs/5.1/components/buttons/#icon-only. Play and pause icons seem to be too small.

![Screenshot from 2022-04-19 09-06-12](https://user-images.githubusercontent.com/17381666/163945633-748d0fa0-f134-471c-b7b8-fe091dad26d3.png)